libqtile/widget/sensors.py

```diff
@@ -89,21 +88,24 @@ def get_temp_sensors(self):
                         kernel_module if kernel_module else "UNKNOWN", str(empty_index)
                     )
                     empty_index += 1
-                temperature_list[label] = (str(round(sensor.current, 1)), unit)
+                temperature_list[label] = sensor.current
 
         return temperature_list
 
     def poll(self):
         temp_values = self.get_temp_sensors()
         if temp_values is None:
             return False
-        text = ""
-        if self.show_tag and self.tag_sensor is not None:
-            text = self.tag_sensor + ": "
-        text += "".join(temp_values.get(self.tag_sensor, ["N/A"]))
-        temp_value = float(temp_values.get(self.tag_sensor, [0])[0])
+        temp_value = temp_values.get(self.tag_sensor, float("nan"))
```

Comment:
```suggestion
        temp_value = temp_values.get(self.tag_sensor, float("N/A"))
```
I believe this would be more understandable.

Reply:
But in Python, `N/A` cannot be converted to a float number. I'm using `nan` here because it's a float number instead of text now.

Reply:
Users may be a bit confused by the "nan" display.

If we can't get a temperature value then we could just display an error message instead.